RELEASE/scripts/sl_ascend/sl_combat.ash

```diff
@@ -1414,9 +1414,29 @@ string sl_combatHandler(int round, string opp, string text)
 			}
 		}
 
-		if(my_location() == $location[The Smut Orc Logging Camp] && canUse($skill[Stuffed Mortar Shell]) && have_effect($effect[Spirit of Peppermint]) != 0 && canSurvive(1.0))
+		if(my_location() == $location[The Smut Orc Logging Camp] && canSurvive(1.0))
 		{
-			return useSkill($skill[Stuffed Mortar Shell]);
+			// Listed from Most to Least Damaging to hopefully cause Death on the turn when the Shell hits.
+			if(canUse($skill[Stuffed Mortar Shell]) && have_effect($effect[Spirit of Peppermint]) != 0)
+			{
+				return useSkill($skill[Stuffed Mortar Shell]);
+			}
+			else if(canUse($skill[Saucegeyser]))
+			{
+				return useSkill($skill[Saucegeyser]);
+			}
+			else if(canUse($skill[Saucecicle]))
+			{
+				return useSkill($skill[Saucecicle]);
+			}
+			else if(canUse($skill[Cannelloni Cannon]) && have_effect($effect[Spirit of Peppermint]) != 0)
+			{
+				return useSkill($skill[Cannelloni Cannon]);
+			}
+			else if(canUse($skill[Northern Explosion]))
+			{
+				return useSkill($skill[Northern Explosion]);
+			}
```

Comment:
log a "oh no, we didn't have anything to use" message in red here please

Reply:
Ah Yes, that is better for Debugging. Added.

Reply:
This one line should stay red.